content/social-login/index.md

```diff
@@ -1,188 +1,122 @@
-
-
-There are also [consumer trends findings](https://drive.google.com/file/d/1CGZyunO2qSTutcq0STcPkzrJ8ypJ8v28/view?pli=1) indicating that around **70% of 18-25 year-olds prefer social login methods** over standard login. While this percentage actually goes down as age increases, the following reasons are surely going to further boost the prevalence of social logins.
-
-
-### Reasons for implementing Social login  
-
-- **Quick profile setup**: Social login pulls in user information from social media profiles, allowing users to quickly complete their profiles without manually entering their data. This streamlined onboarding helps users get started with your app more quickly. 
-
-- **Personalize the user experience**: Social login allows apps to access a user's social profile data (with their permission), which can be used to personalize the user experience and personalized experiences are often more engaging and can lead to higher user retention.
-
-Overall, social login offers a smooth, secure, and personalized user experience that encourages continued engagement with your platform.
-
-
-## Technical Implementation of Social Logins
-After discussing the importance of social logins, the next step is to dive deep into how to implement them with [SuperTokens](https://supertokens.com/docs/thirdparty/custom-ui/thirdparty-login) while highlighting their unique features. 
-
-First, you need to decide how you want users to log into your website or mobile application. With Supertokens, there are various user recipes available to choose from when implementing social logins. These include email and password with Social Login, [passwordless with Social Login](https://supertokens.com/docs/thirdpartypasswordless/introduction), and social login only.
-
-Next, you should then proceed to know the flows that the third-party login providers support and these are:
-
-**Flow 1: Via authorization code (one-time use code) - for web and mobile apps.**
-
-This flow is typically used for web and mobile apps and is split into two branches:
-- **Configured Client Secret**: The client secret is set up on the backend.
-- **No Configured Client Secret**: The client secret is not provided on the backend.
+### Apple
 
-**Flow 2: Via OAuth / access tokens and is implemented for mobile apps.**
+Apple’s “Sign in with Apple” is focused on privacy and security. It gives users the option to hide their email addresses and creates unique, app-specific IDs.
 
-For their explanation go check out [SuperTokens documentation on social login](https://supertokens.com/docs/thirdparty/custom-ui/thirdparty-login).
+Refer to their [official documentation](https://developer.apple.com/sign-in-with-apple/get-started/) to learn more.
 
-## Best practices
+### Twitter
```

Comment:
Change this to X?

Reply:
Done, renamed the title to X (Formerly twitter) just to keep Twitter as a possible search keyword for the blog